kratos/geometries/line_2d_2.h

```diff
@@ -792,6 +792,31 @@ class Line2D2 : public Geometry<TPointType>
     }
 
     /**
+     * It computes the normal of the geometry, if possible
+     * @return The normal of the geometry
+     */
+    virtual array_1d<double, 3> Normal() override
```

Comment:
That depends of each geometry, that's why it should be defined in the specific geometries if needed. For example project in not plane geometries is a complex subject too.